code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -840,6 +781,9 @@
 		help_shake_act(user)
 		return FALSE
 
+	/*else if(istype(W, /obj/item/weapon/storage/bag/gadgets/part_replacer))
```

Comment:
?

Reply:
can remove if unatomic

Reply:
It's not clear to me why you are removing the part replacer behavior, is this just depreciated behavior or...?